code/modules/events/maintenance_predator_vr.dm

```diff
@@ -0,0 +1,73 @@
+/datum/event/maintenance_predator
+	startWhen = 1
+	announceWhen = 50
+	endWhen = 100
+	var/announceProb = 35
+	var/list/possible_mobs = list("Space Bumblebee" = /mob/living/simple_mob/vore/bee,
+								  "Voracious Lizard" = /mob/living/simple_mob/vore/aggressive/dino,
+								  "Giant Frog" = /mob/living/simple_mob/vore/aggressive/frog,
+								  "Giant Rat" = /mob/living/simple_mob/vore/aggressive/rat,
+								  "Crate Mimic" = /mob/living/simple_mob/vore/aggressive/mimic,
```

Comment:
If we add the player as a mimic, let's make sure we do two things.

One I will mention here because it specifically pertains to mimics, delete any remaining mimics that may or may not be on the map.

Reply:
That'll be the ones to the east of the surface armoury in the hidden room

Reply:
Resolved by removing the mimic from the list. Its way too close to the morph in nature, which is separate event, and I do think that some in-wall caches with mimics both now and in future should stay as well.